### PR TITLE
Clarifications to Checkout instructions language around etherpad

### PR DESCRIPTION
**Pull request by wmotion**
This contribution is for the checkout exercise in the instructor training.
I hope this helps

### CODE REVIEW

**Comment by wmotion** [comment]
The correct message for commit dbc62d6 is "Checkout.md: add hyphens in compound words"

File: _extras/checkout.md [hunk 1]
```diff
@@ -219,13 +219,14 @@ think your lesson has good start points for participatory instruction, you may e
 work for a checkout demo. Be sure to give yourself time to change course if your planned lesson will not work.
 
 ### Sign-up and Set-up
-To sign up, select a session that works for you on the [the teaching demo schedule Etherpad]({{page.demopad}}), and add
-your name and a link to your lesson of choice to the Etherpad. Be sure to **double check the time in your local time zone** by clicking on the converter
+To sign up, select a session that works for you on [the Training Demonstration Sessions Etherpad]({{page.demopad}}), and add
+your name and a link to your lesson of choice to that Etherpad. Be sure to **double check the time in your local time zone** by clicking on the converter
```
Comment (brownsarahm) [inline _extras/checkout.md:223]:
```suggestion
To sign up, select a session that works for you on [the Teaching Demo Etherpad]({{page.demopad}}), and add
your name and a link to your lesson of choice to an open slot. Be sure to **double check the time in your local time zone** by clicking on the converter
```
We should use the name here that matches, as best as possible how trainers will refer to that pad, and what it it titled.

File: _extras/checkout.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ meetings, CarpentryCon, and CarpentryConnect events are all eligible to satisfy
 
 ### Getting Credit
 After each Community Discussion meeting, hosts submit a form that includes the names of attendees who identified themselves as
-fulfilling their checkout requirement. This is indicated by writing "checkout" next to your name in the Etherpad. As long as you
+fulfilling their checkout requirement. This is indicated by writing "checkout" next to your name when you register in the Community Discussion Etherpad. As long as you
```
Comment (brownsarahm) [inline _extras/checkout.md:177]:
```suggestion
fulfilling their checkout requirement. This is indicated by writing "checkout" next to your name in the Community Discussion Etherpad when you register. As long as you
```
swapping order of clauses for readability

**Review by karenword — CHANGES_REQUESTED** [review]
Thanks, this looks very helpful! I suggested expanding the title of the demos page a little (I hadn't noticed that this pad had been retitled!) -- in our communications "training" refers to "instructor training" vs more general teaching so keeping these together is necessary.  I think the intent of retitling that pad was probably to make it clear that demos are for instructor training purposes only (unlike Community Discussions).

File: _extras/checkout.md [hunk 3]
```diff
@@ -219,13 +219,14 @@ think your lesson has good start points for participatory instruction, you may e
...
-To sign up, select a session that works for you on the [the teaching demo schedule Etherpad]({{page.demopad}}), and add
-your name and a link to your lesson of choice to the Etherpad. Be sure to **double check the time in your local time zone** by clicking on the converter
+To sign up, select a session that works for you on [the Training Demonstration Sessions Etherpad]({{page.demopad}}), and add
+your name and a link to your lesson of choice to that Etherpad. Be sure to **double check the time in your local time zone** by clicking on the converter
 link posted. Also, examine the demo description to ensure that it is not a special session targeting a specific sub-community or
 language (unless you are part of that target group).
 
-The link to connect to the video conference is at the top of the Etherpad. Your Trainer may send an email with confirmation and/or
-additional instructions prior to your demo. If possible, plan to use a headset with built in microphone for the best audio experience.
+The link to connect to the video conference is at the top of the Traning Demonstration Sessions Etherpad.
```
Comment (brownsarahm) [inline _extras/checkout.md:227]:
```suggestion
The link to connect to the video conference is at the top of the Teaching Demos Etherpad. 
```
The word teaching missing from the title was misleading, I've corrected it on the etherpad and this should match

File: _extras/checkout.md [hunk 4]
```diff
@@ -126,12 +126,12 @@ community.
 
 When signing up on the [Community Discussion Etherpad][discussion], be sure to examine the description of your event carefully. Also, **take care to double check
 the time zone**, and mark your calendar after signing up! At the time of the event, the Zoom link may be found at the top
-of the Etherpad.
+of that Etherpad.
```
Comment (brownsarahm) [inline _extras/checkout.md:129]:
```suggestion
of the discussion Etherpad.
```
Since the goal is clarity, let's go all the way to a specific term

**Review by brownsarahm — CHANGES_REQUESTED** [review]
Thanks @wmotion this caught some important clarity /reference issues.  I have proposed some further edits.

**Comment by brownsarahm** [comment]
Oh @karenword and I overlapped in reviewing this... we should figure out what we want to do, I didn't see hers post while I was working.

File: _extras/checkout.md (hunk 2 above)
Comment (wmotion) [reply _extras/checkout.md:177]: This change is unequivocal and can be accepted.

File: _extras/checkout.md (hunk 4 above)
Comment (wmotion) [reply _extras/checkout.md:129]: If the terms has to be specific, be it 'Community Discussion Etherpad' or 'Discussion Etherpad'. A new specification within the same paragraph let one wonder whether it goes about a new object and, in fact, it doesn't. I suggest no change.

File: _extras/checkout.md (hunk 1 above)
Comment (wmotion) [reply _extras/checkout.md:223]: Re line 222, the current title of the document is "The Carpentries Instructor Training Teaching Demonstration Sessions". "Teaching Demo Etherpad" is confusing until that becomes the title of the document.

Re line 223, makes sense.

File: _extras/checkout.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (wmotion) [reply _extras/checkout.md]: Re line 227, the current title of the document is "The Carpentries Instructor Training Teaching Demonstration Sessions". "Teaching Demo Etherpad" is confusing until that becomes the title of the document.

**Review by wmotion — COMMENTED** [review]
There is a point of terminology as to the title of the document linked to at https://pad.carpentries.org/teaching-demos. Its current title is "The Carpentries Instructor Training Teaching Demonstration Sessions". I maintain that this document should be referred to by its title, which can be changed of course.

File: _extras/checkout.md (hunk 3 above)
Comment (wmotion) [reply _extras/checkout.md]: Re line 227, the current title of the document is "The Carpentries Instructor Training Teaching Demonstration Sessions". "Teaching Demo Etherpad" is confusing until that becomes the title of the document.

**Review by karenword — APPROVED** [review]

**Pull request merged**